### PR TITLE
UpdateNaming

### PR DESCRIPTION
**Pull request by CharlesLee01**

### CODE REVIEW

**Comment by kouyk** [comment]
Closing since it increases confusion for users.

**Comment by CharlesLee01** [comment]
Wait, if u look at the edit command it's s/SESSION. Unless you want to change it to DATETIME_OF_SESSION

**Comment by kouyk** [comment]
If you're keen on editing, change it to `SESSION_DATETIME` instead, drop the commit that changed it to `SESSION` as well. Do change the title of your PR so I won't accidentally close it.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: page
 title: User Guide
 ---
 * Table of Contents
-{:toc}
+  {:toc}
```
Comment (kouyk) [inline docs/UserGuide.md:6]: Don't touch this line, it is not meant to be indented.

Comment (CharlesLee01) [reply docs/UserGuide.md:6]: OK, will change it. It's autocorrected by intellij and I didn't realise.

**Review by kouyk — APPROVED** [review]

**Pull request merged**